components/compliance-service/scanner/db.go

```diff
@@ -219,9 +219,14 @@ func (s *Scanner) UpdateResult(ctx context.Context, job *types.InspecJob, output
 		} else {
 			result.Result = fmt.Sprintf("unable to complete scan on node. please check the logs for more information. node id ref: %s", job.NodeID)
 		}
-		connectionErr := ""
+		var connectionErr, errMsg string
 		if inspecErr != nil {
-			connectionErr = inspecErr.Type
+			if len(inspecErr.Message) > 300 {
+				errMsg = inspecErr.Message[0:300]
```

Comment:
should we append "..." to the end of an error message instead of truncating it to make it apparent that there was more info that followed?

Reply:
mmm, good point! i'll update!